packages/ui-extensions/docs/surfaces/point-of-sale/reference/components/Stack.doc.ts

```diff
@@ -37,91 +37,92 @@ The following examples will demonstrate some, but not all of the abilities of th
       ),
-      image: 'extension-stack-horizontal-flexChildren.png',
+      image: 'extension-stack-inline-flexChildren.png',
     },
     {
       type: 'Generic',
-      anchorLink: 'horizontal-center-children',
-      title: 'Horizontal Stack with centered children',
+      anchorLink: 'inline-center-children',
+      title: 'Inline Stack with centered children',
       sectionContent: `
-You can also center elements in your \`horizontal\` stack. For this, you can specify the \`alignment\` to be \`center\`. However, in this case you also want \`flexChildren\` to be \`false\` (which is the default), so that the children can take up the minimal amount of space that they need, and be centered.
+You can also center elements in your \`inline\` stack. For this, you can specify the \`alignment\` to be \`center\`. However, in this case you also want \`flexChildren\` to be \`false\` (which is the default), so that the children can take up the minimal amount of space that they need, and be centered.
 `,
       codeblock: generateCodeBlockForStack(
-        'Horizontal Stack with centered children',
-        'horizontal-center-children',
+        'Inline Stack with centered children',
+        'inline-center-children',
       ),
-      image: 'extension-stack-horizontal-centered.png',
+      image: 'extension-stack-inline-centered.png',
     },
     {
       type: 'Generic',
-      anchorLink: 'horizontal-end-children',
-      title: 'Horizontal Stack with children at the end',
+      anchorLink: 'inline-align-items-center',
+      title:
+        'Inline Stack with nested block stack and single child that are center aligned on the main axis',
       sectionContent: `
-To make the children be aligned to the end of your \`horizontal\` container, you just need to specify \`alignment\` to be \`flex-end\`. Note that in the first example, the children were at the start of the container. This is because the default value is \`flex-start\`.
+To make the children be aligned to the end of your \`inline\` container, you just need to specify \`alignment\` to be \`flex-end\`. Note that in the first example, the children were at the start of the container. This is because the default value is \`flex-start\`.
 `,
       codeblock: generateCodeBlockForStack(
-        'Horizontal Stack with children at the end',
-        'horizontal-end-children',
+        'Inline Stack with children at the end',
+        'inline-align-items-center',
       ),
-      image: 'extension-stack-horizontal-flex-end.png',
+      image: 'inline-align-items-center.png',
     },
     {
       type: 'Generic',
-      anchorLink: 'vertical',
-      title: 'Vertical Stack',
+      anchorLink: 'block',
+      title: 'Block Stack',
       sectionContent: `
-You can specify your \`Stack\` to layout its children vertically by setting \`direction\` to \`vertical\`.
+You can specify your \`Stack\` to layout its children vertically by setting the \`direction\` prop to \`block\`.
```

Comment:
some food for thought is that referring to these things as props is a react-specific detail